fastify.js

```diff
@@ -581,6 +590,11 @@ function wrapRouting (httpHandler, { rewriteUrl, logger }) {
   }
 }
 
+function loadVersion () {
+  const pkg = JSON.parse(fs.readFileSync(path.join(__dirname, 'package.json')))
```

Comment:
Is there a reason why it makes more sense to read and parse manually rather than require it? Do we want to avoid caching it for some reason?

Reply:
Yes. To avoid keeping the "large" object in memory.

Reply:
require involves caching the required file